src/Validation/TextPolicyValidator.php

```diff
@@ -21,6 +24,25 @@ class TextPolicyValidator {
 	// Could use an URL validation strategy
 	const CHECK_URLS_DNS = 2;
 
+	public function __construct( StringList $badWords = null, StringList $whiteWords = null ) {
```

Comment:
Could have used `Iterator` as well... that is more fuzzy, though has a lot of basic implementations already, avoiding the need for things such as `ArrayBasedStringList`. #SighPHP